pkg/controller/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -153,6 +181,15 @@ func (r *ReconcileMachineHealthCheck) Reconcile(request reconcile.Request) (reco
 			mhc.Spec.MaxUnhealthy,
 			totalTargets-currentHealthy,
 		)
+		r.recorder.Eventf(
+			mhc,
+			corev1.EventTypeWarning,
+			EventRemediationRestricted,
+			"Remediation restricted due to exceeded number of unhealthy machines (total: %v, unhealthy: %v, maxUnhealthy: %v)",
+			totalTargets,
+			mhc.Spec.MaxUnhealthy,
+			totalTargets-currentHealthy,
```

Comment:
Is this correct? Aren't the values for current and max unhealthy swapped here?

Reply:
You're right. Fixed